cylc/flow/cfgspec/suite.py

```diff
@@ -913,8 +913,8 @@
                    Date-time when task job starts running
                 ``%(finish_time)s``
                    Date-time when task job exits
-                ``%(user@host)s``
-                   user@host where the task job is submitted
+                ``%(platform_name)s``
```

Comment:
some debugging in this file can be removed ... lines 1330 & 1331

Reply:
Sorry not related to this pr but noticed it and thought might as well tidy as we go?

Reply:
Nice spot. And since this is a PR to a branch which is itself a PR, you've probably just saved me responding to this comment from Hilary....